modevo-consistency/src/main/java/giis/modevo/consistency/Oracle.java

```diff
@@ -0,0 +1,184 @@
+package giis.modevo.consistency;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import com.datastax.oss.driver.api.core.cql.BoundStatementBuilder;
+import com.datastax.oss.driver.api.core.cql.ColumnDefinition;
+import com.datastax.oss.driver.api.core.cql.PreparedStatement;
+
+import giis.modevo.migration.script.ScriptException;
+import giis.modevo.migration.script.execution.CassandraConnection;
+
+/**
+ * Main class of the oracle that contains the methods that execute the database statements to obtain the projections from the 
+ * SQL database and the data from each Cassandra table and the methods that verify data integrity in the Cassandra database
+ */
+public class Oracle {
+	
+	/**
+	 * Main oracle method to determine if the Cassandra database maintains data integrity
+	 * @return True if the databases are storing same data, false otherwise
+	 */
+	public boolean oracleCompare(Map<String, String> tableQuery, String keyspaceCassandra, String properties,
+			Connection con) {
+		OracleCSV csv = new OracleCSV();
+		Iterator<String> keyIterator = tableQuery.keySet().iterator();
```

Comment:
> There are other places were iterator is used for maps

@javiertuya I found them in other modules from past code. Were you referring to those? In the consistency module I think I changed them to iterate over entrySet

Reply:
@suarezgpablo I was refering this module because there were "unusual" operations passing iterators as parameters. Do a simple search for `.iterator()`, I find 5 references in the consistency module.